src/AppKit/NSCollectionViewDelegate.cs

```diff
@@ -18,9 +20,9 @@ public virtual NSDragOperation ValidateDrop (NSCollectionView collectionView, NS
 	public partial class NSCollectionViewDelegateFlowLayout {
 		[Mac (10, 11)]
 		[Obsolete ("Use 'ValidateDropOperation (NSCollectionView collectionView, NSDraggingInfo draggingInfo, ref NSIndexPath proposedDropIndexPath, ref NSCollectionViewDropOperation proposedDropOperation)' instead.")]
-		public virtual NSDragOperation ValidateDrop (NSCollectionView collectionView, NSDraggingInfo draggingInfo, out NSIndexPath proposedDropIndexPath, out NSCollectionViewDropOperation proposedDropOperation)
+		public virtual NSDragOperation? ValidateDrop (NSCollectionView collectionView, NSDraggingInfo draggingInfo, out NSIndexPath proposedDropIndexPath, out NSCollectionViewDropOperation proposedDropOperation)
```

Comment:
This is a breaking change, since NSDragOperation is an enum.

Reply:
Tha tis a typo. Went too fast.